<launch>
  <node pkg="exbotxi_avoidence" type="avoidence_base" name="avoidence_base" output="screen">
    <param name="limit_range1" value="0.3"/>
    <param name="limit_range2" value="0.8"/>
    <param name="limit_points1" value="10"/>
    <param name="limit_points2" value="15"/>
    <param name="angle_speed" value="0.6"/>
    <param name="scan_in" value="scan"/>
    <param name="vel_in" value="vel_in"/>
    <param name="vel_out" value="cmd_vel"/>
  </node>
</launch>
